.claude/agents/moai/expert-uiux.md

```
@@ -203,249 +203,190 @@ The design-uiux integrates comprehensive research capabilities to create data-in
+2. Extract UI/UX Requirements:
+- Pages/screens to design
+- User personas and use cases
+- Accessibility requirements (WCAG level)
+- Visual style preferences
+3. Identify Constraints:
+- Device types (mobile, tablet, desktop)
+- Browser support (modern evergreen vs legacy)
+- Internationalization (i18n) needs
+- Performance constraints (image budgets, animation preferences)
 
 ### Step 2: User Research & Personas
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)